docs/src/content/docs/blog/introducing-lunaria.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Introducing Lunaria
+excerpt: 
+  Lunaria is a complete localization management toolchain built especially for open-source projects. Currently, Lunaria is already being used in production to help track over 5000 content files in over a dozen of locales in big projects such as [Astro](https://i18n.docs.astro.build/), [Starlight](https://i18n.starlight.astro.build/), [SolidJS](https://docs.solidjs.com/i18n-status/), and [VitePress](https://vitepress.dev/_translations/).
+date: 2024-05-21
+authors:
+  name: Yan Thomas
+  title: Maintainer
+  picture: https://avatars.githubusercontent.com/u/61414485
+  url: https://github.com/yanthomasdev/
+---
+
+Maintaining a project for an international audience is no easy task!
+
+It is a lot of effort to build a sustainable workflow for content to be progressively and consistently localized, especially if you also have contributors to your project.
+
+Lunaria is a complete **localization management toolchain** built especially for open-source projects. 
+
+Lunaria is already being used in production to help track over 5,000 content files in over a dozen locales in big projects such as [Astro](https://i18n.docs.astro.build/), [Starlight](https://i18n.starlight.astro.build/), [SolidJS](https://docs.solidjs.com/i18n-status/), and [VitePress](https://vitepress.dev/_translations/).
+
+:::tip[Ready to get started?]
+Read the official ["Getting Started" guide](https://lunaria.dev/getting-started/) to learn more about and set up Lunaria in your project. 
+:::
+
+## Why Lunaria?
+
+While making the Astro documentation available for translation, we noticed how hard it was to build a sustainable workflow for both maintainers and contributors. Most localization management platforms (the ones that help you keep track of and update localizations) have the requirement of owning your content and are paid to some degree. More importantly, these platforms are way too complex for beginners or might not even be available for people in certain countries one might want to reach.
+
+With all this in mind, the Astro maintainer [`@hippotastic`](https://github.com/hippotastic) built a tailor-made translation tracking tool that allowed us to work with contributors to translate the official Astro documentation into 14 different languages with ease.
```

Comment:
Similar to my comment before, weird to say "our" here since Lunaria != Astro and this isn't a blog post on the Astro blog, I guess changing a bit fixes this?

```suggestion
Wanting a truly free, open-source solution that integrated seamlessly into the translators' and maintainers' normal workflow, the Astro maintainer [`@hippotastic`](https://github.com/hippotastic) built a tailor-made translation tracking tool. The community contributors used this to organize and mobilize, translating the official Astro documentation into 14 different languages with ease.
```